source/server/management/performance-tweaks.rst

```diff
@@ -0,0 +1,92 @@
+==================
+Performance Tweaks
+==================
+
+Sponge provides several performance enhancing and tweaking options to help you run a lag-free server, even
+under heavy load.
+
+.. note::
+  While we try to improve the performance of every server, the performance gains depend on your setup. Please run some
+  tests to ensure you configured your Sponge server to get the most out of it!
+
+Entity Activation Range
+=======================
+
+This setting will alter the loading of entities around players. Lowering the value will only load close entities, while
+raising it will also load entities that are far away from the player. Lower this to improve your servers performance,
+especially with high entity and player counts.
+
+.. code-block:: none
+
+  entity-activation-range {
+        # If enabled, newly discovered entities will be added to this config with
+        # a default value.
+        auto-populate=false
+
+        # Default activation ranges used for all entities unless overidden.
+        defaults {
+            ambient=32
+            aquatic=32
+            creature=32
+            misc=16
+            monster=32
+        }
+
+Lighting Patch to Ignore Unloaded Chunks
+========================================
```

Comment:
In terms of descriptive text, you may be interested in https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/mixin/optimization/world/MixinWorldServer_Lighting.java#L63.